interfaces/builtin/opengl.go

```diff
@@ -114,6 +114,9 @@ unix (bind,listen) type=seqpacket addr="@cuda-uvmfd-[0-9a-f]*",
 # https://github.com/Xilinx/XRT/tree/master/src/runtime_src/core/edge/drm
 /sys/devices/platform/amba{,_pl@[0-9]*}/amba{,_pl@[0-9]*}:zyxclmm_drm/* r,
 
+# Imagination PowerVR driver
```

Comment:
could you provide a link to some document from Imagination that at least lists this name?

Reply:
I didn't find any documents from Imagination official website. Have to register with imagination to get documents. 
Here is all information I can get: [1].  PowerVR GE8300[2] is the chip on MTK i300a board.
Not sure if [1] is enough for you.
[1] https://www.imaginationtech.com/powervr-gpu/ge8300/
[2] https://www.mediatek.com/products/AIoT/mt8362a
If it's okay to you, I'll send a PR to add [1] into opengl.go

Reply:
no, it's fine. Did you test that this is all that is needed to make things work?

Reply:
yes. I verified it by instructions[1] provided by mir team.
[1] https://bugs.launchpad.net/baoshan/+bug/1929173/comments/8